frontend/resourceadm/components/ImportResourceModal/ServiceContent/ResourceContent/ResourceContent.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { ReactNode, useState } from 'react';
+import classes from './ResourceContent.module.css';
+import { Altinn2LinkService } from 'app-shared/types/Altinn2LinkService';
+import { useTranslation } from 'react-i18next';
+import { ResourceNameAndId } from 'resourceadm/components/ResourceNameAndId';
+
+export type ResourceContentProps = {
+  altinn2LinkService: Altinn2LinkService;
+  resourceIdExists: boolean;
+};
+
+/**
+ * @component
+ *    Displays the Resource content in the import resource from Altinn 2 modal
+ *
+ * @property {Altinn2LinkService}[altinn2LinkService] - The service to import from
+ * @property {boolean}[resourceIdExists] - If the id already exists
+ *
+ * @returns {ReactNode} - The rendered component
+ */
+export const ResourceContent = ({
+  altinn2LinkService,
+  resourceIdExists,
+}: ResourceContentProps): ReactNode => {
+  const { t } = useTranslation();
+
+  const [id, setId] = useState(altinn2LinkService.serviceName);
+  const [title, setTitle] = useState(altinn2LinkService.serviceName);
+
+  const [editIdFieldOpen, setEditIdFieldOpen] = useState(false);
+  const [bothFieldsHaveSameValue, setBothFieldsHaveSameValue] = useState(true);
+
+  /**
+   * Replaces the spaces in the value typed with '-'.
+   */
+  const handleIDInput = (val: string) => {
+    setId(val.replace(/\s/g, '-'));
```

Comment:
I see that this regex is used in multiple places. Should we consider creating utils for replaceWhiteSpaceWithHypens? 🤔